website/integrations/services/actual-budget/index.mdx

```diff
@@ -89,7 +89,7 @@ Alternatively, it is possible to configure OpenID Connect via the UI.
       5. Scroll up and click **Start using OpenID** under the **Authentication method** section.
       6. Fill in the following values:
           - **OpenID Provider**: authentik
-          - **OpenID provider URL**: <kbd>https://<em>authentik.company</em>/application/o/<em>your-application-slug</em>/</kbd>
+          - **OpenID provider URL**: `https://authentik.company/application/o/your-application-slug/`
```

Comment:
Don;t we want the variable in angle brackets and underscores? So `<your_application_slug>`?